src/main/java/blackjack/view/InputView.java

```diff
@@ -0,0 +1,36 @@
+package blackjack.view;
+
+import blackjack.utils.StringUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    public static final Scanner scanner = new Scanner(System.in);
+
+    private InputView() {}
+
+    public static List<String> getUsersName() {
+        System.out.println(StringUtils.GET_USERS_NAME_MESSAGE);
+        String input = scanner.nextLine();
+        return Arrays.stream(input.split(","))
+                .map(String::trim)
+                .collect(Collectors.toList());
```

Comment:
이런 형태도 가능할 거 같은데 어떨까요??
```suggestion
        return Arrays.asList(input.split(","));
              
```

Reply:
`trim`을 수행하고 싶어서 이렇게 해봤는데, 다은님이 추천해주신 것처럼 간결하게 작성해보고 싶네요.
제가 기존에 작성된 방법보다 간결하면서 `trim`도 수행할 수 있는 방법이 뭐가 있을까요?

Reply:
사실 보배님이 작성하신 코드와 별 차이가 안나는 거 같긴한데
Arrays.asList(input.replaceAll("\\p{Z}", "").split(",")) 
이런 방식도 있을 거 같아요ㅎㅎㅎ

Reply:
공백을 제거하는데 정규표현식이 있었군요 ! 
찾아보니 컴퓨터마다 캐릭터셋이 다른 경우가 있다보니, 문자열 공백제거할 땐 다은님이 작성하신 정규표현식을 사용한 공백제거법이 좋다고 하네요 ! 감사합니다 👍 
Ref) https://selfinvestfriends.tistory.com/19